site/app/views/AutoGradingView.php

```diff
@@ -387,7 +387,7 @@ public function showTAResults(TaGradedGradeable $ta_graded_gradeable, bool $regr
         $gradeable_id = $gradeable->getId();
         $id = $this->core->getUser()->getId();
         if($gradeable->isTeamAssignment()){
-            $id = $this->core->getQueries()->getTeamByGradeableAndUser($gradeable_id, $id)->getId();
+            $id = $teamID;
```

Comment:
If you're already doing this check in the controller, you can literally just pass $id in to the view, and you don't need the if statement, and you don't have to do any assignment to $id simplifying the code. You don't want this same logic in multiple places. I would recommend leaving it in the controller.